xcode/Subconscious/Shared/Components/OmniboxView.swift

```diff
@@ -64,15 +64,30 @@ struct OmniboxSlashlinkView: View {
     
     var body: some View {
         HStack(spacing: 0) {
```

Comment:
No change from previous. Tried to minimize what I touched. In general, though, I have a tendency to return a single top-level view, since in earlier versions of SwiftUI I had trouble with switches that returned heterogeneous view types.

I think this has since been resolved, so probably just superstition on my part.